pkg/capi/components/CCVariables/index.vue

```diff
@@ -0,0 +1,251 @@
+<script lang="ts">
+import debounce from 'lodash/debounce';
+import { defineComponent } from 'vue';
+import type { PropType } from 'vue';
+import { randomStr } from '@shell/utils/string';
+import { ClusterClassVariable } from '../../types/clusterClass';
+import type { CapiClusterVariable } from '../../types/cluster.x-k8s.io.cluster';
+import { isDefined } from '../../util/validators';
+import Variable from './Variable.vue';
+
+export default defineComponent({
+  name: 'ClusterClassVariables',
+
+  components: { Variable },
+
+  props: {
+    clusterClass: {
+      type:    Object,
+      default: () => {}
+    },
+
+    // <cluster.x-k8s.io>.spec.topology.variables
```

Comment:
is this intentional?

Reply:
Originally yeah, but I guess it's redundant now that the cluster variables are typed. I've removed it